docs/CLI.md

```diff
@@ -52,64 +54,59 @@ cov versions
 cov serve
 ```
 * Starts up the ng server with needed flags
-* Same as `ng serve --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true`
 
 ```
-cov serve-e2e
+cov serve-prod
 ```
 * Starts the ng server with flags needed to run e2e coverage
-* Same as `node --max_old_space_size=5048 ./node_modules/.bin/ng serve --aot --prod --sourcemap=true --build-optimizer --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true --aot --prod --optimization`
```

Comment:
why did you remove the memory from this documentation?

Reply:
Oops.